src/pages/index.js

```diff
@@ -66,20 +108,85 @@ function renderCard(data) {
 /*-------------------------------------------------------------------------------*/
 
 function handleProfileFormSubmit(data) {
+  profileEditPopup.setLoading(true);
   userInfo.setUserInfo(data);
```

Comment:
`userInfo.setUserInfo(data);`

You need to update the `DOM` only in `then` blocks after a successful response from the server, so that when there is an error, it shouldn’t change

**Please, correct it everywhere in the code**